editor/reducer.js

```diff
@@ -472,7 +481,10 @@ export function blockInsertionPoint( state = {}, action ) {
 export function preferences( state = PREFERENCES_DEFAULTS, action ) {
 	switch ( action.type ) {
 		case 'TOGGLE_SIDEBAR':
-			return {
+			return action.isMobile ? {
```

Comment:
For my own curiosity, I tried a different syntax to avoid 2 objects :)
```js
const isSidebarOpenedKey = action.isMobile ? 'isSidebarOpenedMobile' : 'isSidebarOpened';
return {
    ...state,
    [ isSidebarOpenedKey ]: ! state[ isSidebarOpenedKey ],
};
```

It might be a bit harder to read, but sharing with you anyway :)